src/components/views/rooms/EntityTile.tsx

```diff
@@ -133,7 +133,7 @@ export default class EntityTile extends React.PureComponent<IProps, IState> {
             }
             nameEl = (
                 <div className="mx_EntityTile_details">
-                    <div className="mx_EntityTile_name" dir="auto">
+                    <div className="mx_EntityTile_name">
```

Comment:
Specifying the direction on this level forces child elements to be moved to the right side even on the LTR layout, if the user name includes characters in a RTL language. `DisambiguatedProfile` should take care of the direction.